tools/build_chain.sh

```diff
@@ -346,7 +360,7 @@ gen_cert_secp256k1() {
         cp "${agpath}/../ca.crt" "$certpath/"
     fi
 
-    rm -f $certpath/${type}.csr $certpath/${type}.pubkey $certpath/${type}.param
+    rm -f $certpath/${type}.csr $certpath/${type}.pubkey $certpath/secp256k1.param
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5503649)